copy.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs");
+
+fs.cp("src/assets/images", "lib/assets/images", { recursive: true }, (err) => {
+  if (err) {
+    throw new Error(err);
+  } else {
+    console.log("successfully copied files");
+  }
+});
```

Comment:
Questionable implementation but better than a new dependency lol

Reply:
symlinks require privilage, and most projects tend to copy files to avoid inconsistencies.

Reply:
Not sure what you mean by privilage. You don't need sudo access to symlink.

Reply:
was referring to windows, linux by default doesn't need it (may vary depending on the distro/policy)